codex-rs/core/src/config.rs

```diff
@@ -314,27 +315,37 @@ pub fn write_global_mcp_servers(
         for (name, config) in servers {
             let mut entry = TomlTable::new();
             entry.set_implicit(false);
-            entry["command"] = toml_edit::value(config.command.clone());
+            match &config.transport {
+                McpServerTransportConfig::Stdio { command, args, env } => {
+                    entry["command"] = toml_edit::value(command.clone());
+
+                    if !args.is_empty() {
```

Comment:
if it is empty, should we make sure it is not defined on `entry`?

Reply:
transport is flattened into entry so that's how it's defined in the toml